aws/environments/oauth-sync.yml

```diff
@@ -8,3 +8,27 @@ owner: "rfkelly@mozilla.com"
 reaper_spare_me: "true"
 content_docker_tag: feature.oauth-sync
 sync_docker_tag: feature.oauth-sync
+
+# N.B. the leading newline on the value below is important,
+# it prevents ansible from parsing it as a python dict.
```

Comment:
😭

Reply:
Literal hours of banging-head-on-desk involved in discovering this behaviour...